packages/transformer/src/IModelExporter.ts

```diff
@@ -448,13 +439,13 @@ export class IModelExporter {
       return;
     }
 
-    const initOpts: ExporterInitOptions =
-      typeof accessTokenOrOpts === "object"
-        ? accessTokenOrOpts
-        : {
-            accessToken: accessTokenOrOpts,
-            startChangeset: { id: startChangesetId },
-          };
+    const startChangeset =
+      "startChangeset" in args! ? args.startChangeset : undefined;
```

Comment:
Instead of using ! can we just check if its defined before checking "startchangeset" in args